definitions/ext-datadog_system/golden_metrics.yml

```diff
@@ -0,0 +1,19 @@
+systemLoad:
+  title: System Load
+  query:
+    select: sum(system.load.1), sum(system.load.5), sum(system.load.15)
```

Comment:
remember that golden metrics are the most important metrics to look at.
In this case which of the 3 is the most important one? (maybe load.15 or load.5?)

I would say the same for the rest.
CPU we normally look into usage not system, iowait, etc...

Reply:
The golden metrics here were given as a whole by the authors of the ticket, so I will relay the feedback.  I will also see if they would be ok changing the multiple value metrics into dashboards instead of golden metrics.